buildenv/jenkins/common/pipeline-functions.groovy

```diff
@@ -345,7 +345,11 @@ def build_with_slack(DOWNSTREAM_JOB_NAME, ghprbGhRepository, ghprbActualCommit,
         // Set Github Commit Status
         if (ghprbActualCommit) {
             node(SETUP_LABEL) {
-                set_build_status(GITHUB_REPO, DOWNSTREAM_JOB_NAME, ghprbActualCommit, DOWNSTREAM_JOB_URL, 'SUCCESS', "Build PASSED")
+                try {
+                    retry_and_delay(set_build_status(GITHUB_REPO, DOWNSTREAM_JOB_NAME, ghprbActualCommit, DOWNSTREAM_JOB_URL, 'SUCCESS', "Build PASSED")
```

Comment:
I think you need to wrap your command in `{ curly braces }`

Reply:
I also think we should be adding this to all the places where we run `set_build_status`

Reply:
Great idea. Thanks for checking!

Reply:
Hey @AdamBrousseau! I changed all the function calls. Thanks!